src/ripple/overlay/impl/PeerImp.cpp

```diff
@@ -100,6 +102,20 @@ PeerImp::run()
     if(! strand_.running_in_this_thread())
         return strand_.post(std::bind (
             &PeerImp::run, shared_from_this()));
+    {
+        auto s = getVersion();
+        if (boost::starts_with(s, "rippled-"))
```

Comment:
Consider `static std::string const prefix("rippled-");`. This lets us use the `size` in `erase` instead of `8`. Obviously correct as written, your call.